app/controllers/availabilities_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class AvailabilitiesController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @time_ranges = TimeRange.order(:start_time)
+    @availabilities = load_availabilities
+    @start_time = start_time
+    @end_time = end_time
+  end
+
+  def update_availabilities
+    availability_ids = if params.key?(:availability_ids)
+                         params[:availability_ids] else []
+                       end
+    Availability.where(user_id: current_user.id).each do |availability|
+      set(availability,
+          availability_ids.include?(availability.id.to_s))
+    end
+  end
+
+  private
+
+  def set(availability, status)
```

Comment:
If `status` is a boolean this can be changed to `availability.update(status: status)`, making it more concise. That's why I was thinking if we just change that to boolean instead of enum